### PR TITLE
Add man, readme, and license files to packages

### PR DESCRIPTION
**Pull request by foxzi**
* add man and readme files to server package
* add license files to packages
* add dependency for dev/devel package

Follows up #8771 
Closes https://github.com/tarantool/infra/issues/188

### CODE REVIEW

**Review by ylobankov — CHANGES_REQUESTED** [review]

File: static-build/cmake/MakePackages.cmake [hunk 1]
```diff
@@ -38,8 +38,33 @@ install(DIRECTORY tarantool-prefix/share/man/man1 DESTINATION share/man/
...
+        DESTINATION /usr/share/doc/tarantool
+        COMPONENT server)
+
 set(CPACK_GENERATOR "DEB;RPM")
 
+if (PACKAGE_TYPE STREQUAL "DEB")
+    install(FILES ../debian/copyright
+            DESTINATION /usr/share/doc/tarantool
+            COMPONENT server)
+    install(FILES ../LICENSE
```
[the remark this answers is not in the document]
Comment (foxzi) [reply static-build/cmake/MakePackages.cmake:51]: I'm not sure, but It doesn't brake rules. I checked packages on my ubuntu desktop and find that it's ok. If You want, I can remove it from deb packages.

File: static-build/cmake/MakePackages.cmake [hunk 2]
```diff
@@ -38,8 +38,33 @@ install(DIRECTORY tarantool-prefix/share/man/man1 DESTINATION share/man/
...
+             DESTINATION /usr/share/doc/tarantool
+             COMPONENT server)
+    install(FILES ../AUTHORS
+             DESTINATION /usr/share/doc/tarantool
+             COMPONENT server)
+endif()
+
+if (PACKAGE_TYPE STREQUAL "RPM")
+    install(FILES ../LICENSE
+            DESTINATION /usr/share/license/tarantool
```
[the remark this answers is not in the document]
Comment (foxzi) [reply static-build/cmake/MakePackages.cmake]: fixed

File: static-build/cmake/MakePackages.cmake [hunk 3]
```diff
@@ -38,8 +38,33 @@ install(DIRECTORY tarantool-prefix/share/man/man1 DESTINATION share/man/
...
         FILES_MATCHING PATTERN "tarantool.1")
 
+install(FILES ../README.md
+        DESTINATION /usr/share/doc/tarantool
+        COMPONENT server)
+
 set(CPACK_GENERATOR "DEB;RPM")
 
+if (PACKAGE_TYPE STREQUAL "DEB")
+    install(FILES ../debian/copyright
```
[the remark this answers is not in the document]
Comment (foxzi) [reply static-build/cmake/MakePackages.cmake]: done

File: static-build/cmake/MakePackages.cmake (hunk 1 above)
[the remark this answers is not in the document]
Comment (ylobankov) [reply static-build/cmake/MakePackages.cmake:51]: If it is ok, let's keep it.

**Review by ylobankov — APPROVED** [review]

**Pull request merged**